app/controllers/steps/appeal/must_wait_challenge_decision_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Steps::Appeal
+  class MustWaitChallengeDecisionController < Steps::AppealStepController
```

Comment:
Can we name this a bit different? Maybe put a "for" in there – `MustWaitForChallengeDecision`?